### PR TITLE
fix(css): adjust css distribution and add minification

### LINKED ISSUE #432: Package updates - modify css distribution and add minification
Modify our dist to make the following changes in our npm package:

patternfly.css will be full css (everything)  (this was patternfly-full.css before)
patternfly.min.css will be everything, but minified
patternfly-base.css - this will be only the bare minimum needed for building a site if you are pulling in individual components

### PR DESCRIPTION
**Pull request by dgutride**
Closes #432

### CODE REVIEW

File: src/patternfly/patternfly-base.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import "./utilities/_all";
+@import "./_fonts";
+@import "./_icons";
```
Comment (dmiller9911) [inline src/patternfly/patternfly-base.scss]: `@patternfly/react-core` will be using SVGs for icons and wont need the `./_icons`.  would it be possible to remove this from here and place it in the full version? Fonts should be ok to keep.

Comment (mindreeper2420) [reply src/patternfly/patternfly-base.scss]: @dmiller9911 that file is necessary to pull in the Font Awesome styling. Without it, the icons will not be available in anything but the full version.

**Comment by matthewcarleton** [comment]
@dgutride looks like the styles for the workspace is not loading :(

File: src/patternfly/patternfly-base.scss (hunk 1 above)
Comment (dmiller9911) [reply src/patternfly/patternfly-base.scss]: Ok. I will write something to remove this at build when we copy over the styles.

**Comment by dgutride** [comment]
@matthewcarleton  - should be working now, was referencing the full file instead of the base one

**Pull request merged**